.github/workflows/version.yaml

```diff
@@ -37,11 +38,21 @@ jobs:
         run: yarn install --immutable
       - name: Prepare
         run: yarn prepare
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@1a317071222a9bfb1839df1b58b1f0dcd893b589
```

Comment:
@633kh4ck could u have a look at this?

Reply:
I gave this a review and I can't see any malicious attempts at trying to steal the keys. Note that I didn't review the `openpgp` package which has access to the private key here: https://github.com/crazy-max/ghaction-import-gpg/blob/60f6f3e9a98263cc2c51ebe1f9babe82ded3f0ba/src/openpgp.ts#L18-L20

Reply:
good to merge for now? the consequence of that GPG key leaking wouldn't be dramatic imo

Reply:
ping @633kh4ck

Reply:
why do we need a thirdparty action for this / why can't this be a command?